src/Template/Layout/js/app/pages/modules/view.js

```diff
@@ -48,7 +48,11 @@ export default {
             event.stopPropagation();
             const form = new FormData(event.target);
             const action = event.target.getAttribute('action');
-            const response = await fetch(`${action}Json`, {
+            let endpoint = action;
+            if (!endpoint.endsWith('/save')) {
+                endpoint = `${action}Json`;
+            }
+            const response = await fetch(`${endpoint}`, {
```

Comment:
I think it's safe to just use the form's action (see [here](https://github.com/bedita/manager/blob/master/src/Template/Pages/Modules/view.twig#L21)) and remove this `if`. This Vue component is used only in the linked template, so there shouldn't be any drawbacks

```suggestion
            const response = await fetch(action, {
```